content/appeals/decision-reviews/release-notes/2023-09-04.md

```diff
@@ -0,0 +1,12 @@
+New endpoints have been added to allow downloads of submitted PDFs. When you submit a Higher-Level Review, or a Notice of Disagreement, a Supplemental Claim, you can now access a watermarked copy of the PDF as submitted to the VA with the following caveats:
+
+1. A PDF download will become available only after the appeal has progressed to the 'submitted' state.
+2. The PDF will stop being available one week after the appeal has progressed to the 'completed' state (when the veteran's personally identifying information is purged from our severs).
+3. PDFs are only available for appeals created with an associated veteran ICN, which is provided in the `X-VA-ICN` header when the appeal is first created. If the appeal was not created with an `X-VA-ICN` header, a PDF will never become available.
+
+The PDF download endpoints are:
+* `/services/appeals/v2/decision_reviews/higher_level_reviews/ID/download`
```

Comment:
Will the asterisks render as bullet points in prod?

Reply:
If prod uses standard-compliant markdown, then yes, but just to be safe, I'll copy what I'm seeing in a few other release notes from before, where dashes were used instead